integrationtests/elasticsearch-indexmanager-it/start-elastic.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=version.elasticsearch | grep -v '\[.*\]')
```

Comment:
I did not know about ```mvn elasticsearch:run```, I'll add the the README.txt.
I also like the docker approach that makes it very easy to install new plugins